src/controllers/GitHubController.ts

```diff
@@ -0,0 +1,74 @@
+import ProjectData, { ProjectMember } from '../models/ProjectData';
+import * as Octokit from "@octokit/rest";
+
+type RepoType = "public" | "private" | "forks" | "all";
+
+export default class GitHubControler {
+    org: string;
+    type: RepoType;
+    octokit: Octokit;
+
+    constructor(authToken: string, org?: string, type?: RepoType) {
+        this.org = org || "aggie-coding-club";
+        this.type = type || "public";
+        
+        this.octokit = new Octokit({auth: authToken});
+    }
+    
+    getProjectData(): Promise<ProjectData[]> {
+        return this.octokit.repos
+            .listForOrg({
+                org: this.org,
+                type: this.type
+            })
+            .then((reposResponse) => {
+                const repos = reposResponse.data;
+                let projectList: ProjectData[] = [];
+
+                for (let repo of repos) {
+                    let project: ProjectData = {
+                        repoURL: repo.html_url,
+                        name: repo.name,
+                        description: repo.description
+                    };
+                    projectList.push(project);
+                }
+
+                return projectList;
+            }).then(projectList => {
+                let apiCalls = [];
```

Comment:
You're not updating the memory location here, just use `const`.

```suggestion
                const apiCalls = [];
```